### PR TITLE
Check if dupe is in admin queue

### PR DESCRIPTION
**Pull request by ThaTiemsz**
This will return a message if someone does the !dupe command and the suggestion was already reported as a dupe.

Additionally I added a force flag one can use, which skips the message, cause maybe it's useful I don't know ¯\\\_(ツ)_/¯. I can remove it just as easily, if you guys think it's unneeded. Works as follows:

`!dupe https://feedback.discordapp.com/forums/326712-discord-dream-land/suggestions/16782913-video-calling http://feedback.discordapp.com/forums/326712-discord-dream-land/suggestions/10521852-add-video-chat -f`

### CODE REVIEW

File: Commands/Commands/queue.js [hunk 1]
```diff
@@ -635,6 +638,17 @@ commands.registerVote = {
   }
 }
 
+function dupeDupe (uvid) {
+  if (uvid === null) return false
+  r.db('DFB').table('queue').filter({ type: 'adminMergeRequest', UV1: uvid }).run().then(results => {
+    if (results >= 1) {
```
Comment (Dougley) [inline Commands/Commands/queue.js]: `if (results > 0)`

**Review by Dougley — CHANGES_REQUESTED** [review]

File: Commands/Commands/queue.js [hunk 2]
```diff
@@ -635,6 +638,17 @@ commands.registerVote = {
   }
 }
 
+function dupeDupe (uvid) {
+  if (uvid === null) return false
+  r.db('DFB').table('queue').filter({ type: 'adminMergeRequest', UV1: uvid }).run().then(results => {
+    if (results > 0) {
```
Comment (Dougley) [inline Commands/Commands/queue.js]: `results.length` 👀

**Review by Dougley — CHANGES_REQUESTED** [review]

**Comment by ThaTiemsz** [comment]
🙄

**Review by Dougley — APPROVED** [review]
👀

**Pull request merged**